src/js/api/auth/login.mjs

```diff
@@ -0,0 +1,30 @@
+import { apiSocialUrl } from "../constants.mjs";
+import * as storage from "../../storage/storageHandler.mjs"
+
+const action = "/auth/login";
+const method = "post";
+
+export async function login (profile) {
+    const loginUrl = apiSocialUrl + action;
+    const body =  JSON.stringify(profile);
+
+   const response = await fetch (loginUrl, {
+        headers: {
+            "Content-Type": "application/json"
+        },
+        method,
+        body
+    });
+
+    const result = await response.json();
+
+    // Check if login is successful and redirect if it is, else alert login failed.
+    if (response.ok) {
+        storage.save("token", result.accessToken);
+        storage.save("profile", result)
```

Comment:
```js
const { accessToken, ...profile } = await response.json();
```

This is a good opportunity to use object destructuring, which is a requirement in the CA